lib/ukstore/include/uk/store.h

```diff
@@ -301,10 +373,14 @@ _uk_store_get_folder(unsigned int library_id, const char *name);
 struct uk_store_entry *
 _uk_store_get_dynamic_entry(unsigned int libid, const char *f_name,
 				const char *e_name);
+struct uk_store_entry *
+_uk_store_get_static_entry(unsigned int libid, const char *e_name);
 
 static inline struct uk_store_entry *
 _uk_store_get_entry(unsigned int libid, const char *f_name, const char *e_name)
 {
+	if (!f_name)
```

Comment:
You should also add checks on the release/free functions when it is a static entry. delete should fail, release can be totally ignored since we do not do refcounting.